com.github.hopsan.Hopsan.json

```diff
@@ -0,0 +1,75 @@
+{
+    "app-id": "com.github.hopsan.Hopsan",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.12",
+    "sdk": "org.kde.Sdk",
+    "command": "hopsangui",
+    "modules": [
+        {
+            "name": "doxygen",
+            "buildsystem": "cmake",
+            "config-opts": [ "-DCMAKE_BUILD_TYPE=Release" ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/doxygen/doxygen/archive/Release_1_8_15.tar.gz",
+                    "sha256": "cc5492b3e2d1801ae823c88e0e7a38caee61a42303587e987142fe9b68a43078"
+                }
+            ],
+            "cleanup": [ "*" ]
+        },
+        {
+            "name": "hopsan",
+            "disabled": false,
+            "buildsystem": "simple",
+            "build-commands": [
+                /* Configure source code with version numbers and build settings */
+                "./buildDebPackage/prepareSourceCode.sh $(pwd) $(pwd) 2.11.5 20190430.2145 2.11.5.20190430.2145 false false",
+                /* Since Hopsan already has built-in dependency module build scripts we reuse them instead of duplicating the work with flatpak modules */
+                "cd Dependencies; ./setupDiscount.sh",
+                "cd Dependencies; ./setupFMILibrary.sh",
+                "cd Dependencies; ./setupHDF5.sh",
+                "cd Dependencies; ./setupKatex.sh",
+                "cd Dependencies; ./setupZeroMQ.sh",
+                "cd Dependencies; ./setupQwt.sh",
+                /* Build Hopsan */
+                "qmake HopsanNG.pro",
+                "make -j$(nproc)",
+                /* Install Hopsan reusing deb packaging script */
+                "./buildDebPackage/copyInstallHopsan.sh $(pwd) /app"
+            ],
+            "post-install": [
+                "install -t /app/share/icons/hicolor/128x128/apps/ -Dm644 com.github.hopsan.Hopsan.png",
+                "install -t /app/share/applications/ -Dm644 com.github.hopsan.Hopsan.desktop",
+                "install -t /app/share/appdata/ -Dm644 com.github.hopsan.Hopsan.appdata.xml"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/Hopsan/hopsan.git",
+                    "tag": "v2.11.5-flatpak",
+                    "commit": "0ac99dc8fa92dce2eb2dfaf87eff20c6bd8ecab0"
+                },
+                {
+                    "type": "file",
+                    "path": "com.github.hopsan.Hopsan.png"
+                },
+                                {
+                    "type": "file",
+                    "path": "com.github.hopsan.Hopsan.desktop"
+                },
+                {
+                    "type": "file",
+                    "path": "com.github.hopsan.Hopsan.appdata.xml"
+                }
+            ]
+        }
+    ],
+    "finish-args": [
```

Comment:
Those should be at the top of the manifest. Just before modules.

Reply:
fixed